packages/php-storage-driver-common/proto/bucket.proto

```diff
@@ -25,6 +25,7 @@ message CreateBucketCommand {
   }
 
   message CreateBucketBigqueryMeta {
+    option deprecated = true;
```

Comment:
Hm pride mi, ze chceme deprekovat len ten region, ale takto deprekujes celu tu metu, https://github.com/keboola/storage-backend/pull/138/files#diff-cba0b541d7c54768da361b8094bdff7a67c1b80b98ecc45151f4e2e4b1e87efdR12 mozno sa mylim ale mozno casom na nieco budeme potrebova tu metu co na to @keboola/connection

Reply:
Este to prosim uprav https://keboolaglobal.slack.com/archives/C0553EM2RGX/p1711550127180529?thread_ts=1711524086.972039&cid=C0553EM2RGX

Reply:
ok, upravil jsem